### PR TITLE
Update covid19v2 flairs

### PR DESCRIPTION
**Pull request by ebxn**
Updates covid's flair images and filenames

### CODE REVIEW

File: app/views/players/show.html.haml [hunk 1]
```diff
@@ -15,6 +15,8 @@
       = image_tag "flairs/for_ulven.png"
     - elsif @player.player_name == "Tame My Wild"
       = image_tag "flairs/tame.png"
+    - elsif @player.player_name == "Covid 19 V2"
+      = image_tag "flairs/covid_19_v2.png"
```
Comment (vmeow) [inline app/views/players/show.html.haml:19]: Thanks for the help! This player has another `elsif` clause around line ~194, could we merge the two please?

Comment (ebxn) [reply app/views/players/show.html.haml:19]: Absolutely, I'll get right on it!

**Comment by vmeow** [comment]
@ebxn breh
Screen Shot 2020-07-18 at 12 16 15 PM

**Comment by vmeow** [comment]
Closing for PR #87 which re-sizes virus and resolves conflicts.

**Pull request closed without merging**